packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -296,15 +300,14 @@ const SigninContainer = ({
       let result = await trySignIn(
         email,
         v1Credentials,
-        error ? undefined : v2Credentials,
-        unverifiedAccount,
+        v2Credentials,
```

Comment:
There is one fewer argument now.  Was there a bug here previously?  😕

Reply:
No bug here. That argument was removed from the function as well. The unverified account isn't important anymore, since the check on if the session is verified should now suffice.

Reply:
Whoa I didn't see it because Github collapsed that single line of diff by default.  WTH

Reply:
Oh sorry, I think I just missed it.  🤦 😭